.changeset/thirty-dancers-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@crossmint/client-sdk-verifiable-credentials": minor
+---
+
+Added crossmint encryption
```

Comment:
this file can be removed, not sure why it carried over :thinking:

Reply:
Can you do so in your branch?